src/java.base/share/classes/java/lang/Double.java

```diff
@@ -749,7 +882,7 @@ public static String toHexString(double d) {
      * @return     a {@code Double} object holding the value
      *             represented by the {@code String} argument.
      * @throws     NumberFormatException  if the string does not contain a
-     *             parsable number.
```

Comment:
Was `parsable number` supposed to be deleted?

Reply:
My mistake in editing; I'll correct it. Thanks.